### PR TITLE
LPS-173505 Fix bug by implementing DDMExpressionFieldAccessor

### PR DESCRIPTION
**Pull request by devpedrotavares**
Related issue: [LPS-173505](https://issues.liferay.com/browse/LPS-173505)

### CODE REVIEW

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/validation/rule/DDMObjectValidationRuleEngineImpl.java [hunk 1]
```diff
@@ -52,6 +53,10 @@ public Map<String, Object> execute(
 				_ddmExpressionFactory.createExpression(
 					CreateExpressionRequest.Builder.newBuilder(
 						script
+					).withDDMExpressionFieldAccessor(
+						new ObjectEntryDDMExpressionFieldAccessor(
+							(Map<String, Object>)inputObjects.get(
+								"objectEntry"))
```
Comment (carolmariaabb) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/internal/validation/rule/DDMObjectValidationRuleEngineImpl.java:59]: @p-albuquerque @devpedrotavares please remember to rename here when this [commit](https://github.com/liferay-objects/liferay-portal/pull/1688/commits/23c0698c7d4699283dc7aff001e683b1b217ff41) is merged

**Review by carolmariaabb — APPROVED** [review]
@devpedrotavares Great job! Please read my comment below.

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/validation/rule/DDMObjectValidationRuleEngineImpl.java (hunk 1 above)
Comment (p-albuquerque) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/internal/validation/rule/DDMObjectValidationRuleEngineImpl.java:59]: Understood, thank you.

**Comment by devpedrotavares** [comment]
ci:test:relevant

**Comment by devpedrotavares** [comment]
ci:test:object

**Comment by devpedrotavares** [comment]
ci:reevaluate:1347401_3456

**Comment by devpedrotavares** [comment]
forwarding directly: https://github.com/brianchandotcom/liferay-portal/pull/129694

**Comment by devpedrotavares** [comment]
ci:close

**Pull request closed without merging**